src/formatters/utils/sortResults.ts

```diff
@@ -1,13 +1,79 @@
 import { IRuleResult } from '../../types';
 
-export const sortResults = (results: IRuleResult[]) => {
-  return [...results].sort((resultA, resultB) => {
-    const diff = resultA.range.start.line - resultB.range.start.line;
+const compareCode = (one: string | number | undefined, another: string | number | undefined): number => {
```

Comment:
```suggestion
const compareCode = (left: IDiagnostic['code'], right: IDiagnostic['code']): -1 | 0 | 1 => {
```
Not sure why, but I like left and right as names for such comparison functions. :laughing: 
Feel free to leave it as is though.

Reply:
@P0lip Fixed